lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1949,6 +1949,9 @@ definitions:
       nat_enabled:
         type: boolean
         example: true
+      enable_converged_core:
+        type: boolean
+        example: false
```

Comment:
Add a description - what are the implications of turning this on, or leaving it off?

Reply:
Comment Added.